src/components/organisms/teacher/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from "./OakSearchFilterCheckBox";
 export * from "./OakTertiaryOLNav";
+export * from "./OakUnitListItem";
```

Comment:
Is there a lint rule for import order? Wondering why this wasn't already in the right place

Reply:
my script auto generates the exports for the index.ts files so overwrites the old versions

Reply:
Ahh yes it's export I see now!

Reply:
If I put it in the ci/cd then I will make it ordered to prevent confusing changes